partitioned-table.md

```diff
@@ -193,11 +193,13 @@ PARTITION BY RANGE COLUMNS(name,valid_until)
 
 It will split the data by year and by name in the ranges [''-'G'), ['G'-'M'), ['M'-'S'), ['S',), allowing you to easily drop invalid data while still benefit from partition pruning on both `name` and `valid_until` columns.
 
+It will partition the data by year and by name in the ranges ['', 'G'), ['G', 'M'), ['M', 'S') and ['S',). It allows you to easily drop invalid data while still benefit from partition pruning on both `name` and `valid_until` columns. In this example, `[ , ]` is a closed range, and `[,)` is a half-open and half-closed range.
```

Comment:
```suggestion
It will partition the data by year and by name in the ranges ['', 'G'), ['G', 'M'), ['M', 'S') and ['S',). It allows you to easily drop invalid data while still benefit from partition pruning on both `name` and `valid_until` columns. In this example, `[,)` is a half-open, half-closed range. For example, ['G', 'M') indicates a range containing `G` and from `G` to `M`, but excluding `M`.
```

Reply:
What about "a left-closed, right-open range"
See https://en.wikipedia.org/wiki/Interval_(mathematics)#:~:text=An%20interval%20is%20said%20to%20be%20left%2Dopen%20if%20and,%2Dclosed%20and%20right%2Dopen.